nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/reporting/AbstractReportingTaskNode.java

```diff
@@ -72,6 +77,27 @@ public AbstractReportingTaskNode(final ReportingTask reportingTask, final String
         this.reportingTask = reportingTask;
         this.processScheduler = processScheduler;
         this.serviceLookup = controllerServiceProvider;
+
+        final Class<?> reportingClass = reportingTask.getClass();
+
+        try {
+            if (reportingClass.isAnnotationPresent(DefaultSchedule.class)) {
+                DefaultSchedule dsc = reportingClass.getAnnotation(DefaultSchedule.class);
+                try {
+                    this.setSchedulingStrategy(dsc.strategy());
+                } catch (Throwable ex) {
+                    LOG.error(String.format("Error while setting scheduling strategy from DefaultSchedule annotation: %s", ex.getMessage()), ex);
+                }
+                try {
+                    this.setSchedulingPeriod(dsc.period());
+                } catch (Throwable ex) {
+                    this.setSchedulingStrategy(SchedulingStrategy.TIMER_DRIVEN);
+                    LOG.error(String.format("Error while setting scheduling period from DefaultSchedule annotation: %s", ex.getMessage()), ex);
+                }
+            }
+        } catch (Throwable ex) {
+            LOG.error(String.format("Error while setting default schedule from DefaultSchedule annotation: %s",ex.getMessage()),ex);
```

Comment:
Wondering based on the code above and internal try/catch is it even possible to get here?